packages/web-components/src/components/ic-chip/ic-chip.tsx

```diff
@@ -40,6 +42,12 @@ export class Chip {
    */
   @Prop() appearance?: IcChipAppearance;
 
+  /**
+   * The custom chip colour. This will only style the chip component if dismissible=false.
+   * Can be a hex value e.g. "#ff0000", RGB e.g. "rgb(255, 0, 0)", or RGBA e.g. "rgba(255, 0, 0, 1)".
+   */
+  @Prop() customColor?: IcColor = null;
```

Comment:
Would be good to add a sentence on how this overrides the transparentBackground prop/attribute.

Also good to add the reverse to the transparentBackground prop

Reply:
I don't think it does override the transparentBackground prop. The transparentBackground prop only affects the outlined variant, and for the outlined variant the customColor prop only changes the border colour.

Reply:
Ahh sorry, I misunderstood what the `transparentBackground `prop is used for